File: src/main/tests/input/const_member.cpp
#include <iostream>

class Example {
private:
    int value;
public:
    Example(int v) : value(v) {}
    int getValue() const noexcept { return this->value; }
};

int main() {
    Example e(100);
    std::cout << "getValue() = " << e.getValue() << "\n";
}

File: src/main/tests/pythonGenerated/const_member.py
import typing

class Example:
    def __init__(self, v):
        self.value = v
    
    def getValue(self) -> int:
        return self.value

def main() -> int:
    e = Example(100)
    print("getValue() = ", e.getValue(), "\n", sep="", end="")

if __name__ == "__main__":
    main()
